tests/contrib/django/test_connection.py

```diff
@@ -30,7 +26,7 @@ def test_connection(self):
 
         span = spans[0]
         eq_(span.name, 'sqlite.query')
-        eq_(span.service, 'defaultdb')
+        eq_(span.service, 'django-defaultdb')
```

Comment:
@palazzem this settings only solve `default` database alias. And if I have two projects that have `read-replica` alias name, we will have the same problem!

What do you think about `settings.DEFAULT_DATABASE_PREFIX` or `settings.DATABASE_PREFIX`?